jobserver/models.py

```diff
@@ -64,13 +64,13 @@ def repo_name(self):
 
 class JobQuerySet(models.QuerySet):
     def completed(self):
-        return self.filter(completed_at__isnull=False)
+        return self.filter(started=True, completed_at__isnull=False)
 
-    def in_progress(self):
-        return self.filter(started_at__isnull=False, completed_at=None)
+    def running(self):
+        return self.filter(started=True, completed_at=None)
 
     def pending(self):
-        return self.filter(started_at=None, completed_at=None)
+        return self.filter(started=False)
```

Comment:
Shouldn't we combine this with the `status_code not in [6, 8]` logic?

Reply:
Yep, I missed that, thanks!